src/swing.ts

```diff
@@ -80,9 +80,10 @@ export interface Stack {
    * Creates an instance of Card and associates it with an element.
    *
    * @param {HTMLElement} element
+   * @param {boolean} prepend
    * @return {Card}
    */
-  createCard(elment: HTMLElement): void;
+  createCard(elment: HTMLElement, prepend: boolean): void;
```

Comment:
@Sparkomatic I think we should have a default here so it doesnt break for people using createCard already